_includes/head.html

```diff
@@ -2,41 +2,56 @@
   <meta charset="utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <title>{% if page.title %}{{ page.title | escape }} - {% endif %}{{ site.title | escape }}</title>
+
+  <title>
+    {% if page.title %}{{ page.title | escape }} - {% endif %}{{ site.title | escape }}
+  </title>
+
   <meta name="description"
     content="{{ page.excerpt | default: site.description | strip_html | normalize_whitespace | truncate: 160 | escape }}" />
 
-  <link rel="stylesheet" href="https://static.opensuse.org/chameleon-2.0/dist/css/chameleon.css" />
+  <!-- stylesheet -->
+  <link rel="stylesheet" href="https://static.opensuse.org/chameleon-3.0/dist/css/chameleon.css" />
   <link rel="stylesheet" href="{{ '/assets/css/app.css' | relative_url }}" />
 
-  <link rel='shortcut icon' type='image/x-icon'
-    href='https://static.opensuse.org/chameleon-2.0/dist/images/favicons/favicon.ico' />
-  <link color="#73ba25" href="https://static.opensuse.org/chameleon-2.0/dist/images/favicons/mask-icon.svg"
-    rel="mask-icon">
-  <link href="{{ '/manifest.json' | absolute_url }}" rel="manifest">
+  <!-- favicon -->
+  <link rel="shortcut icon" type="image/x-icon" href="https://static.opensuse.org/favicon.ico" />
+  <link rel="icon" href="https://static.opensuse.org/favicon-32.png" sizes="32x32">
+  <link rel="icon" href="https://static.opensuse.org/favicon-48.png" sizes="48x48">
+  <link rel="icon" href="https://static.opensuse.org/favicon-64.png" sizes="64x64">
+  <link rel="icon" href="https://static.opensuse.org/favicon-96.png" sizes="96x96">
+  <link rel="icon" href="https://static.opensuse.org/favicon-144.png" sizes="144x144">
+  <link rel="icon" href="https://static.opensuse.org/favicon-192.png" sizes="192x192">
 
+  <!-- apple-touch-icon -->
+  <link rel="apple-touch-icon" href="https://static.opensuse.org/favicon-144.png" sizes="144x144">
+  <link rel="apple-touch-icon" href="https://static.opensuse.org/favicon-192.png" sizes="192x192">
+  <link rel="mask-icon" href="https://static.opensuse.org/mask-icon.svg" color="#73ba25" />
+
+  <!-- mobile web app data -->
+  <link href="{{ '/manifest.json' | absolute_url }}" rel="manifest" />
   <meta name="mobile-web-app-capable" content="yes" />
+  <meta name="theme-color" content="#73ba25" />
 
+  <!-- open graph meta -->
   <meta property="og:site_name" content="{{ site.title | escape }}" />
-  <meta property="og:title"
-    content="{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}" />
-  <meta property="og:type" content="article" />
-  {% if page.layout == 'post' %}
+  <meta property="og:title" content="{{ page.title | default: site.title | escape }}" />
```

Comment:
Semantically, short_title might work better here as the default

Reply:
`title` is `openSUSE News`, `short_title` is `News`. When sharing in third party sites, it is better to include `openSUSE News`, so the audience know which project it is. So I think we can keep `title`.

Reply:
openSUSE News is already a site_name though